### DISCUSSION TITLE
[QUESTION]: multi-level interrupts: single node having interrupts from multiple aggregators

### DISCUSSION THREAD

**Discussion by LaurentiuM1234** [zephyrproject-rtos/zephyr#68771] [General] [answered]
Hi,

Does the current multi-level interrupt architecture support a single node having interrupts from multiple aggregators?
For example, in the following scenario:

```
intc1: interrupt-controller@deadbeef {
    /* some properties go here */
    interrupt-controller;
    #interrupt-cells = <1>;
}

intc1_aggr1: interrupt-controller@cafebabe {
    /* some properties go here */
    interrupt-controller;
    interrupts-extended = <&intc1 19>;
}

intc1_aggr2: interrupt-controller@cafebabe {
    /* some properties go here */
    interrupt-controller;
    interrupts-extended = <&intc1 20>;
}

my_node_label_here: some-dummy-periph@some_dummy_address {
}
```

`my_node_label_here` uses interrupts from both `intc1_aggr1` and `intc1_aggr2`. I wasn't able to get this working so before trying to make any changes I'd like to understand if there's any alternatives to be taken into consideration. Any help would be much appreciated. Thanks!

**Comment by henrikbrixandersen** [comment, marked as answer]
You'll need to use the `interrupts-extended` property for specifying the interrupts in `some-dummy-periph@some_dummy_address`.
